JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -659,16 +636,17 @@
         ANOVAsummary$addFootnote(.mmMessageOmmitedTerms2(removed_te[[i]], names(removed_te)[i]), symbol = gettext("Warning:"))
       }
     }
-    if (length(added_re) > 0) {
-      for (i in 1:length(added_re)) {
+
+    if (length(added_re) > 0)
+      for (i in 1:length(added_re))
```

Comment:
This is a nice use case for `seq_along`. That returns `1:length(added_re)` unless `length(added_re)` is 0, in which case it returns `integer(0)` and the whole body of the loop is never run.
```suggestion
    for (i in seq_along(added_re))
```

Reply:
yes, that's a good point. there is more of these there, I will change them at once